sockeye/beam_search.py

```diff
@@ -305,7 +326,8 @@ def __init__(self,
         self._scorer = scorer
 
     def hybrid_forward(self, F, best_hyp_indices, best_word_indices,
-                       finished, scores_accumulated, lengths, reference_lengths):
+                       finished, scores_accumulated, lengths, reference_lengths,
+                       secondary_factors=None):
```

Comment:
nit: I guess the intention for calling this `secondary_factors` is that the target words are technically the main factor, but given we don't call something primary factor in the arguemnts I'd find just `factors` clearer.

Reply:
ok